reader_core/src/gen7/draw.rs

```diff
@@ -5,7 +5,11 @@ use crate::{
     utils::{format_egg_parent, is_daycare_masuda_method},
 };
 
-pub use crate::draw::{draw_header, draw_pkx};
+const WHITE: u32 = 0xffffff;
+const GREEN: u32 = 0x00cc00;
+const RED: u32 = 0xff0000;
```

Comment:
Let's import the existing colors from `crate::draw`.

Crystal has different colors because it supports less colors than 3ds games, but everything else should share the same colors.